src/gdata/docs/service.py

```diff
@@ -331,7 +331,7 @@ def Download(self, entry_or_id_or_url, file_path, export_format=None,
         url = entry_or_id_or_url
 
     if export_format is not None:
-      if url.find('/Export?') == -1:
+      if (url.find('/Export?') == -1) and (url.find('/export?') == -1):
```

Comment:
Sure. Feel free to do so.

13/03/2015 17:48, Ali Afshar:
>
> In src/gdata/docs/service.py
> <https://github.com/google/gdata-python-client/pull/9#discussion_r26407344>:
>
> > @@ -331,7 +331,7 @@ def Download(self, entry_or_id_or_url, file_path, export_format=None,
> >          url = entry_or_id_or_url
> >  
> >      if export_format is not None:
> > -      if url.find('/Export?') == -1:
> > +      if (url.find('/Export?') == -1) and (url.find('/export?') == -1):
>
> Can we refactor this to something like
> {{{
>
> "/export?" not in url.lower()
>
> }}}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/gdata-python-client/pull/9/files#r26407344>.
>


-- 
Sascha Holzhauer
Diploma in System Science

School of GeoSciences
The University of Edinburgh
Drummond Street, Edinburgh EH8 9XP, UK

Tel +44 (0)131 651 4447
sascha.holzhauer@ed.ac.uk
http://www.geos.ed.ac.uk/homes/sholzhau


The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.